presto-main/src/main/java/io/prestosql/server/InternalCommunicationConfig.java

```diff
@@ -47,6 +48,18 @@ public InternalCommunicationConfig setSharedSecret(String sharedSecret)
         return this;
     }
 
+    public boolean isInternalJwtEnabled()
+    {
+        return internalJwtEnabled;
+    }
+
+    @Config("internal-communication.jwt.enabled")
```

Comment:
Should this be marked as legacy/deprecated? Is there a use case to turn this off? Maybe for debugging? If so, perhaps name it something like `internal-communication.jwt.disabled-for-testing`

(we can merge this as-is for now and follow up later)